nspanel_esphome.yaml

```diff
@@ -936,9 +936,9 @@ number:
     id: temperature_correction
     entity_category: config
     unit_of_measurement: '°C'
-    min_value: -10
-    max_value: 10
-    step: 0.5
+    min_value: -3
```

Comment:
Don't you thing the temperature will deviate more than this new range you are proposing?